src/components/button/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, MouseEvent, ReactNode } from 'react';
+import { Button as ChakraButton } from '@chakra-ui/react';
+
+interface Props {
+  variant?: 'primary';
+  size?: 'md' | 'lg' | 'xl';
```

Comment:
What do you think about giving an explicit name here? Using the names that we will use between Devs and designers

```suggestion
  size?: 'small' | 'regular' | 'large';
```

Reply:
I think this will helps us to understand each other and with the designers as well, when we say `regular` button and so on.

Reply:
I actually followed the Figma token by its name above the button.

<img width="813" alt="image" src="https://user-images.githubusercontent.com/3179244/171818758-ff72de59-a01c-4f2c-bbb8-e5577f000acc.png">

It depends on team's decision whether we choose the full name or shorten name, all fine to me

Reply:
Btw, I agree we should switch size list to `small`/`regular`/`large`.

Reply:
It seems that we have the ROBOT language as winner  -> https://automotivesmg.slack.com/archives/C03GK939L3V/p1654246423987809
We will go with `md, lg, xl`.

Sorry the go and back, but now we will know which naming we will take for the future components. With this decision we will be in sync with designer and devs. 

Thanks you!